IdentityCore/tests/MSIDCurrentRequestTelemetryTests.m

```diff
@@ -0,0 +1,92 @@
+//
+//  MSIDCurrentRequestTelemetryTests.m
+//  IdentityCore
+//
+//  Created by Mihai Petriuc on 5/1/20.
```

Comment:
Please use our default header :)
You can copy it from other project files. There's also a way to configure default headers in Xcode templates but that keeps changing and breaking... so I just normally copy it from other files.

Reply:
Acknowledged and changed everywhere